domain/schema/model/sql/0025-model-status.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE model_status_value (
+    id INT PRIMARY KEY,
+    status TEXT NOT NULL
+);
+
+-- Status values for the model.
+INSERT INTO model_status_value VALUES
+(0, 'error'),
+(1, 'available'),
+-- We set the model status to busy when the model is being upgrading.
+(2, 'busy'),
+-- We set the model status to suspended when the model has invlid cloud credentials.
+(3, 'suspended');
```

Comment:
We need a conversation around if we want this look up value given it's a "virtual" status driven off credential validity and not something we will use in the underlying data model. It's an orthogonal concern to model status and we just happen to currently conflate them in the service layer when it really is more a presentation concern which should be derived from status and credential validity as separate values. 
We can land as is and tweak if necessary.